lua/hbac/telescope/init.lua

```diff
@@ -0,0 +1,93 @@
+local M = {}
+
+local hbac_config = require("hbac.setup").opts
+local make_display = require("hbac.telescope.make_display")
+local state = require("hbac.state")
+local subcommands = require("hbac.command.subcommands")
+local utils = require("hbac.utils")
```

Comment:
I'm noticing I need to move `hbac_config` to `attach_mappings` because otherwise, with the current design, the default opts are being called before the user opts are loaded. I was kind of reckless with just importing everything left and right and need to take a good look at it, but please keep a lookout for anything fishy or something that might cause future problems in this regard. I'm thinking of moving the other hbac module imports to `attach_mappings` too?